blazor/nuget-packages.md

```diff
@@ -1548,6 +1589,7 @@ A package of Blazor popup components Dialog and Tooltip are used to display info
 <td>
 <ul>
 <li><a href="https://blazor.syncfusion.com/documentation/dialog">Dialog</a></li>
+<li><a href="https://blazor.syncfusion.com/documentation/predefined-dialogs/getting-started-webapp">PreDefined Dialog</a></li>
```

Comment:
Predefined => make sure casing

Reply:
Hi @rajendranr-5483 ,

I modified this casing for Predefined Dialog.

Regards,
@Backiaraj